src/brevitas/graph/equalize.py

```diff
@@ -1501,16 +1523,30 @@ def _merge_ln(layer_norm, next_module, scale_bias_by_weight):
 
 class RotationEqualization(GraphTransform):
 
-    def __init__(self) -> None:
+    def __init__(self, blacklist_layers, layers_to_expand) -> None:
         super(RotationEqualization, self).__init__()
+        if blacklist_layers is not None:
+            self.blacklist_layers = blacklist_layers
+        else:
+            self.blacklist_layers = []
+        if layers_to_expand is not None:
+            self.layers_to_expand = layers_to_expand
+        else:
+            self.layers_to_expand = []
+        self.supported_sinks = ()
 
-    def find_module(self, model, regions: List, prefix=''):
+    def find_module(
+            self,
+            model: nn.Module,
+            regions: List[Region],
+            prefix: str = '',
+            blacklist_layers: Optional[List[str]] = None):
```

Comment:
The exclusion list needs to be passed since we want to exclude all the layers that might be expanded